packages/node/src/index.ts

```diff
@@ -20,14 +20,15 @@ import type {
 } from './types'
 import LightMyRequest from 'light-my-request'
 import { ExecutionResult, print } from 'graphql'
+import { platform } from 'os'
 
 function getPinoLogger<TContext, TRootValue>(
   options: YogaNodeServerOptions<TContext, TRootValue>['logging'] = {},
 ): YogaLogger {
   const prettyLog =
     typeof options === 'object' && 'prettyLog' in options
       ? options?.prettyLog
-      : process.env.NODE_ENV === 'development'
+      : process.env.NODE_ENV !== 'production'
```

Comment:
Didn't we decide on opt-in dev mode, default to prod mode?

Reply:
This is only for `prettyLog` because right now logs are in weird json format. I think it is harmless to have pretty logs by default?

Reply:
@ardatan we can also recommend users to add 'pino-pretty' to their dev script

Reply:
Ok removing it for now. We can discuss it later.